ecl/eclccserver/eclccserver.cpp

```diff
@@ -560,12 +641,11 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
             }
             unsigned retcode = pipe->wait();
             errorReader->join();
-            abortWaiter->stop();
```

Comment:
Test by adding a sleep here - and ensure that the compile is aborted.

Reply:
I did... and (in latest version) it was.